docs/cunumeric/source/developer/CONTRIBUTING.md

```diff
@@ -0,0 +1,72 @@
+# Contributing to cuNumeric
```

Comment:
Can this file be linked to the top-level CONTRIBUTING.md, like BUILD.md is?

Reply:
weird I thought I had https://github.com/nv-legate/cunumeric/pull/805/commits/7bb0815e124eef9c4e58ea86cf0d3edc5ce8a5f3